Python/bytecodes.c

```diff
@@ -860,7 +860,7 @@ dummy_func(
             Py_DECREF(v);
         }
 
-        inst(SEND_GEN, (unused/1, receiver, v -- receiver)) {
+        inst(SEND_GEN, (unused/1, receiver, v -- receiver, retval)) {
```

Comment:
Can you try renaming `retval` to `unused`? It might fix the unused variable warnings in `generated_cases.c.h`

Reply:
Done.